From f84b5e61827efb3f08466ce982343e592aa20f82 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Tue, 01 Nov 2022 18:00:48 +0200
Subject: [PATCH] [SW-0] common: correct function argument name

This patch matches argument names of security_rom_get_key_hash_by_idx
function.

Change-Id: I2674036572cd16e6e77de8d26b0e44cd32b77e0b
---

diff --git a/common/security.c b/common/security.c
index 0eccc43..98922ae 100644
--- a/common/security.c
+++ b/common/security.c
@@ -90,14 +90,14 @@
 	return fw_version_sec;
 }
 
-static int security_get_first_valid_key(uint8_t customer)
+static int security_get_first_valid_key(uint8_t key_index)
 {
 	uint8_t end_index, key_revoke;
 
-	/* customer keys are from index 1 - 4
+	/* key_index keys are from index 1 - 4
 	 * habana key is only at index 0
 	 */
-	if (customer)
+	if (key_index)
 		end_index = SECURITY_KEY_HASH_COUNT;
 	else
 		end_index = SECURITY_KEY_HASH_HABANA_COUNT;
@@ -105,9 +105,9 @@
 	key_revoke = sp_sram_readl(SECURITY_EFUSE_KEYS_REVOKE_OFFSET);
 	/* return offset of first non-revoked key
 	 * habana will scan index 0 only
-	 * customer will scan index 1 - 5
+	 * key_index will scan index 1 - 5
 	 */
-	for (uint8_t i = customer; i < end_index; i++) {
+	for (uint8_t i = key_index; i < end_index; i++) {
 		if ((key_revoke & BIT(i)) == 0)
 			return i;
 	}
@@ -116,11 +116,11 @@
 	return -1;
 }
 
-__weak int security_rom_get_key_hash_by_idx(uint32_t *hash, uint8_t customer)
+__weak int security_rom_get_key_hash_by_idx(uint32_t *hash, uint8_t key_index)
 {
 	uint32_t valid_key_base;
 	int i;
-	int valid_key_offset = security_get_first_valid_key(customer);
+	int valid_key_offset = security_get_first_valid_key(key_index);
 
 	if (valid_key_offset < 0)
 		return 0;
diff --git a/common/security.h b/common/security.h
index dc42d99..2f0699f 100644
--- a/common/security.h
+++ b/common/security.h
@@ -118,7 +118,7 @@
 void security_twi_if_rr_hbw_config(uint8_t spi_en);
 void security_i2c_s_enable(void);
 uint8_t security_find_min_fw_version(uint32_t *fw_version);
-int security_rom_get_key_hash_by_idx(uint32_t *hash, uint8_t customer);
+int security_rom_get_key_hash_by_idx(uint32_t *hash, uint8_t key_index);
 int security_check_valid_fw_version(uint32_t fw_version);
 void security_load_revoke_bits(void);
 void security_load_fw_version(void);
